src/cmd-compress

```diff
@@ -116,12 +117,16 @@ def compress_one_builddir(builddir):
             img['uncompressed-size'] = img['size']
             with open(tmpfile, 'wb') as f:
                 t = ncpu()
-                if args.compressor == 'xz':
+                if compressor == 'xz':
                     runcmd(['xz', '-c9', f'-T{t}', filepath], stdout=f)
-                elif args.compressor == 'zstd':
+                elif compressor == 'zstd':
                     runcmd(['zstd', '-10', '-c', f'-T{t}', filepath], stdout=f)
-                else:
+                elif compressor == 'gzip':
                     runcmd(['gzip', f'-{gzip_level}', '-c', filepath], stdout=f)  # pylint: disable=E0606
```

Comment:
noting here that this gzip-level is not the same as the one used previously from `qemuvariants.py`.

Reply:
not sure if we care, just noting the change